src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -350,6 +351,13 @@ object Task {
 
   }
 
+  /** @return a stream of task local volumes, extrapolating them from the app spec */
+  def localVolumes(app: AppDefinition): Iterable[LocalVolume] = {
+    app.container.toSet[Container].flatMap(VolumesModule.localVolumes.collect).map{ volume =>
```

Comment:
can formatting stuff like that be automated? dislike spending human cycles on such